packages/now-cli/src/commands/deploy/latest.js

```diff
@@ -251,7 +251,9 @@ export default async function main(
   if (argv['--name']) {
     output.print(
       `${prependEmoji(
-        `The ${param('--name')} flag is deprecated (https://zeit.ink/1B)`,
+        `The ${param(
+          '--name'
+        )} flag is deprecated (https://vercel.link/name-flag)`,
```

Comment:
Can we call it "option" instead of "flag"? This would match the documentation in the link.

Reply:
Done 👍